feature-libs/customer-ticketing/components/customer-ticketing-list.module.ts

```diff
@@ -12,8 +13,9 @@ import { CustomerTicketingDetailsModule } from './customer-ticketing-details';
     I18nModule,
     FormErrorsModule,
     CustomerTicketingDetailsModule,
+    CustomerTicketingDetailsModule,
   ],
   exports: [],
   providers: [],
 })
-export class CustomerTicketingComponentsModule {}
+export class CustomerTicketingListComponentsModule {}
```

Comment:
👍 , filename should be customer-ticketing-components.module.ts

and you deleted the actual customer-ticketing-list.module.ts and folder (`customer-ticketing-list`) that have the actual listing mapping from backend to frontend (cms component to frontend component)